src/layers/VivViewerLayer/utils.js

```diff
@@ -17,16 +17,15 @@ export function renderSubLayers(props) {
     opacity,
     data,
     colormap,
-    dtype
+    dtype,
+    id
   } = props;
-
   // Only render in positive coorinate system
   if ([left, top, right, bottom].some(v => v < 0)) {
     return null;
   }
-
-  return new XRLayer({
-    id: `XRLayer-left${left}-top${top}-right${right}-bottom${bottom}`,
+  const xrl = new XRLayer({
+    id: `XRLayer-left${left}-top${top}-right${right}-bottom${bottom}-${id}`,
```

Comment:
wont `left`, `top`, `right`, `bottom` be unique to the layer (regardless of viewport id)?

Reply:
unless you're all the way zoomed out?

Reply:
You could have a case where you're at the same zoom level as the overview.  So, I think your second comment captures why this is necessary (also, it makes debugging much easier).

Reply:
sounds good.